### ISSUE TITLE
[Bug]: `setBackgroundMaterial` does not work

### ISSUE THREAD

**Issue by kingyue737** [electron/electron#43345]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

31.1.4

### What operating system(s) are you using?

Windows

### Operating System Version

Windows 23H2 22631.4037

### What arch are you using?

x64

### Expected Behavior

`win.setBackgroundMaterial('mica')` should apply material background effect to window.

### Actual Behavior

Nothing happened. Neither do others like 'tabbed' or 'acylic'

### Testcase Gist URL

https://gist.github.com/kingyue737/d2ca0f1d5318f5915538cb05879bfb81

**Comment by codebytere** [comment]
@kingyue737 yes - it is `win.setBackgroundMaterial('mica')`. I believe you may erroneously have included an extra `set` in the call. If this is not the case, please include a reproducible sample for us to continue triage as we've tested this extensively on Windows and it has worked as expected thus far.

**Comment by kingyue737** [comment]
@codebytere Sorry I forgot the reproduction:  https://gist.github.com/kingyue737/d2ca0f1d5318f5915538cb05879bfb81

As I stated before, the only thing I did was add `mainWindow.setBackgroundMaterial('mica')`.

**Comment by kingyue737** [comment]
To be specific, you can't set the background material of a window if its initial background material is `none` or `undefined`.

And I found if the initial background material of a window is not `none`, even if you set it back to `none` via `setBackgroundMaterial`, it will still trigger the bug mentioned in #42393.

**Comment by Innei** [comment]
Has any solution?

**Comment by kingyue737** [comment]
> Has any solution?
No

The original bug was confirmed #41824 four months ago. I guess it's not easy to fix this.

Then I wanted to find a workaround via `setBackgroundMaterial` but found this issue. Currently, disable either `backgroundMaterial` or `maximizable`

**Comment by kingyue737** [comment]
bump

**Comment by Super1Windcloud** [comment]
The thing is, in most of my templates, mainWindow.setBackgroundMaterial("mica") or mainWindow.setBackgroundMaterial("acrylic") doesn't work.

**Comment by Super1Windcloud** [comment]
@codebytere , can this problem be solved?

**Comment by VIXI0** [comment]
https://github.com/electron/electron/issues/38466#issuecomment-2576225294

here is a workaround

**Comment by kingyue737** [comment]
bump

**Comment by kingyue737** [comment]
bump
